frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyDraggableItem.tsx

```diff
@@ -32,32 +24,18 @@ interface IStrategyDraggableItemProps {
         index: number,
     ) => DragEventHandler<HTMLDivElement>;
     onDragEnd?: () => void;
-}
-
-const onDragNoOp = () => () => {};
+};
 
 export const StrategyDraggableItem = ({
     strategy,
     index,
-    environmentName,
-    otherEnvironments,
     isDragging,
     onDragStartRef = onDragNoOp,
     onDragOver = onDragNoOp,
     onDragEnd = onDragNoOp,
-}: IStrategyDraggableItemProps) => {
-    const projectId = useRequiredPathParam('projectId');
-    const featureId = useRequiredPathParam('featureId');
+    headerItemsRight,
```

Comment:
I'm landing on "headerItemsRight" for now (instead of headerChildren, actions, or other options) because when we get to the playground, we want left-aligned children. I think it might make sense to have a "left" and a "right", but I want to leave that until we get there.